initializr-service/src/main/resources/application.yml

```diff
@@ -783,7 +789,7 @@ initializr:
           artifactId: spring-cloud-starter-oauth2
         - name: Cloud Task
           id: cloud-task
-          description: Task result tracking along with integration with batch and streams
+          description: Task result tracking and integration with Spring Batch
```

Comment:
Unrelated change?!

Reply:
Not really. It focuses the search in the web UI better on stream things for stream apps.